pkg/controller/petset/pet.go

```diff
@@ -249,15 +240,17 @@ func (p *apiServerPetClient) SyncPVCs(pet *pcb) error {
 	errMsg := ""
 	// Create new claims.
 	for i, pvc := range pet.pvcs {
-		_, exists, err := p.getPVC(pvc.Name, pet.parent.Namespace)
-		if !exists {
-			var err error
-			if err = p.createPVC(&pet.pvcs[i]); err != nil {
-				errMsg += fmt.Sprintf("Failed to create %v: %v", pvc.Name, err)
+		_, err := p.getPVC(pvc.Name, pet.parent.Namespace)
+		if err != nil {
+			if errors.IsNotFound(err) {
+				var err error
+				if err = p.createPVC(&pet.pvcs[i]); err != nil {
+					errMsg += fmt.Sprintf("Failed to create %v: %v", pvc.Name, err)
```

Comment:
All errors should start with a lower case.  Realize this is here before but while you are making updates let's fix up